libraries/c_sdk/standard/mqtt/src/iot_mqtt_subscription.c

```diff
@@ -318,70 +299,76 @@ IotMqttError_t _IotMqtt_AddSubscriptions( _mqttConnection_t * pMqttConnection,
                                           const IotMqttSubscription_t * pSubscriptionList,
                                           size_t subscriptionCount )
 {
-    IotMqttError_t status = IOT_MQTT_SUCCESS;
+    IOT_FUNCTION_ENTRY( IotMqttError_t, IOT_MQTT_SUCCESS );
+    /*IotMqttError_t status = IOT_MQTT_SUCCESS; */
```

Comment:
please remove the code in comments if not required

Reply:
Addressed